GitHub_Learning/Feature/Onboarding/Presentation/OnboardingView.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  OnboardingView.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 09/11/2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import CleanCore
+import CleanPlatform
+
+protocol OnboardingView: View {
+    func setPages(_ pages: [OnboardingPageViewModel])
+    func setNumberOfPagesControls(_ controls: Int)
+    func setSelectedPageControlFor(_ index: Int)
+    func setPageTo(_ index: Int)
+    func setButtonTitle(_ title: String)
```

Comment:
Tady by to chtelo trochu upravit cely ten protocol. Jsou veci, ktere spolu souvisi a neni treba je nastavovat 2x. Naprikad `setPages` a `setNumberOfPagesControls`. `setNumberOfPagesControls` ma reprezentovat pocet pages, takze tuto informaci muzeme vzit uz z metody `setPages` a nemusime ji z prezenteru explicitne posilat. 

Stejne tak `setPageTo` a `setSelectedPageControlFor`. Stejny princip.

Navrhuji ten protocol nejak takto:
```suggestion
    func setPages(_ pages: [OnboardingPageViewModel])
    func showPage(at index: Int)
    func setButtonTitle(_ title: String)
```